CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 ### Added
 
 ### Changed
+- Add filters for dataset access permissions and data layers to dashboard's dataset list. Removed `isActive` and `isPublic` columns to save screen space. Switch data layer entries to display layer names instead of categories, e.g. "color" --> "axons". [#6131](https://github.com/scalableminds/webknossos/pull/6131)
```

Comment:
```suggestion
- Various changes to the Dataset table in the dashboard  [#6131](https://github.com/scalableminds/webknossos/pull/6131):
  - Renamed "Allowed Teams" column to "Access Permissions".
  - Add filter functionality to "Access Permissions" column to filter for public datasets.
  - Removed `isActive` and `isPublic` columns to save screen space.
  - Changed data layer entries to display layer names instead of categories, e.g. "color" --> "axons".
```